lib/Doctrine/ORM/Query/QueryException.php

```diff
@@ -204,13 +204,15 @@ public static function overwritingJoinConditionsNotYetSupported($assoc)
     }
 
     /**
+     * @param object $pathExpr
```

Comment:
Is there really nothing more detailed than `object`? :| Sounds like the `SqlWalker` internal API could need a DTO